### ISSUE TITLE
Make possible to install `vlt` using `brew`

### ISSUE THREAD

**Issue by mrazauskas** [vltpkg/vltpkg#233]
Thanks for `vlt`. Looks very promising!

Would you consider making possible to install `vlt` using [Homebrew](https://brew.sh/)?

**Comment by lukekarrys** [comment]
I've never contributed a formula to Homebrew before but if we did do this it would probably be very similar to [`pnpm.rb`](https://github.com/Homebrew/homebrew-core/blob/ac1b806c1b0e6f1b9d69d7a71954394114f8fefd/Formula/p/pnpm.rb) (at least until #153 publishes standalone executables).

**Comment by mrazauskas** [comment]
On second thought, [Corepack](https://nodejs.org/api/corepack.html) or similar solution could be way better than publishing to Homebrew. But that is another story. I close this in favour of Corepack support.

**Comment by lishaduck** [comment]
Given that vlt currently isn't packaged via Corepack and there currently aren't any plans to distribute vlt over corepack (nodejs/corepack#588), can this be re-opened? (I'm happy to have it blocked on #153, I just want a way to install it that's not `npm -g` (or `curl | sh`, for that matter))

**Comment by mrazauskas** [comment]
The problem with this issue is that it only covers macOS. I opened it spontaneously and realised that later. (That’s why Corepack sounded a better solution.)

That said as author of this issue I would like to close it.
